pullrequest.rst

```diff
@@ -37,7 +37,7 @@ Here is a quick overview of how you can contribute to CPython:
 
 #. `Create Pull Request`_ on GitHub to merge a branch from your fork
 
-#. Review and address `comments on your Pull Request`_
+#. Review and address `comments on your Pull Request`_ (without force-pushing)
```

Comment:
With the note, I think the `(without force-pushing)` is unneeded. The `.. note::` directive stands out quite a lot in reST, so I think it will be clear to most readers without effectively repeating the same thing.

Reply:
You are right. I removed it.